docs/CHANGELOG.rst

```diff
@@ -109,6 +109,7 @@ Fixed
 - Ensure Tox installs the package from sdist
 - Pass all Resolwe's environment variables to Tox's testing environment
 - Ensure tests gracefully handle unavailability of Docker
+- Fix reading stdout.txt containing non-ascii characters
```

Comment:
This is inserted at the wrong place.

Also, I think this small change is not worthy of a change log entry.